### ISSUE TITLE
Variation support

### ISSUE THREAD

**Issue by lalop** [habibillah/woocommerce-fixed-quantity#2]
Hy,
I'm looking to build fix functionnality but for variable product.
Do you have any plan to support them ?
If you want I can add this to your repository, let me know.

**Comment by habibillah** [comment]
sure, you may create a PR

**Comment by Phiph** [comment]
Hey guys,

Has @lalop implementation been merged, or it is too bespoke for just variable products. 

Thanks,

**Comment by habibillah** [comment]
Nope, I don't agree with his approach. You may use his fork if you want

**Comment by Phiph** [comment]
Cool thanks. 

Have you guys figured out the best approach - are you working on a new one. 

if not - I'll give it a go!
